test/fixtures/full/terraform.tfvars

```diff
@@ -26,6 +27,7 @@ tables = [
     table_id          = "bar",
     schema            = "sample_bq_schema.json",
     time_partitioning = null,
+    view              = null,
```

Comment:
Requiring all users to set view to null will be a breaking change. Can we avoid this?

Reply:
You are right, it will require everyone to change their existing code. 
I don't know how to make it optional. I have followed a similar design as partitioning option. If you have a suggestion I am open.

Reply:
@umairidris I don't think we can avoid that in current module design. 
@yunus could you pls add upgrading readme file like [that](https://github.com/terraform-google-modules/terraform-google-bigquery/blob/master/docs/upgrading_to_bigquery_v3.0.md)

Reply:
Understood, given we already requrie this for time_partioning then I think it makes sense to follow the same pattern. We got around this in other modules by using the 'any' type e.g. https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/variables.tf#L66 but it also has its own problems, but provides a simpler interface most of the time.

Reply:
Adding a separate `views` parameter instead of collapsing this into the `tables` parameter would also allow backwards compatibility.